src/System.Management.Automation/resources/Modules.resx

```diff
@@ -625,7 +625,7 @@
     <value>The specified module '{0}' was not found. Update the Name parameter to point to a valid path, and then try again. </value>
   </data>
   <data name="WorkflowModuleNotSupportedInPowerShellCore" xml:space="preserve">
-    <value>Cannot load the workflow module '{0}'. Workflow is not supported in PowerShell Core.</value>
+    <value>Cannot load the workflow module '{0}'. Workflow is not supported in PowerShell.</value>
```

Comment:
Put Core back as this is specific to Core

Reply:
Fixed.